code/modules/clothing/outfits/vv_outfit.dm

```diff
@@ -44,113 +44,113 @@
 			r_pocket = item_path
 
 
-/proc/collect_vv(obj/item/I)
+/proc/collect_vv(obj/item/item)
 	//Temporary/Internal stuff, do not copy these.
-	var/static/list/ignored_vars = list("vars","x","y","z","plane","layer","override","animate_movement","pixel_step_size","screen_loc","fingerprintslast","tip_timer")
+	var/static/list/ignored_vars = list("vars","x","y","z","plane","layer","override","animate_movement","pixel_step_size","screen_loc","fingerprintslast","tip_timer","cached_ref","datum_flags")
```

Comment:
Use `NAMEOF()` here so that if we remove one of these variables, it'll throw a compile error, meaning that it's very noticeable and people can continuously keep this list up to date.
```suggestion
	var/static/list/ignored_vars = list(
		NAMEOF(item, animate_movement),
		NAMEOF(item, cached_ref),
		NAMEOF(item, datum_flags),
		NAMEOF(item, fingerprintslast),
		NAMEOF(item, layer),
		NAMEOF(item, override),
		NAMEOF(item, pixel_step_size),
		NAMEOF(item, plane),
		NAMEOF(item, screen_loc),
		NAMEOF(item, tip_timer),
		NAMEOF(item, vars),
		NAMEOF(item, x),
		NAMEOF(item, y),
		NAMEOF(item, z),
	)
```
![image](https://user-images.githubusercontent.com/34697715/211972420-29f7f27e-790c-4b76-bc24-17cdf645c4b9.png)

hey look at that, it's already outdated. be sure to remove those two (and make sure the code works too).

Reply:
Handy.